src/main/java/com/bakdata/kafka/KafkaStreamsApplication.java

```diff
@@ -79,19 +82,19 @@ public abstract class KafkaStreamsApplication extends KafkaApplication implement
     @CommandLine.Option(names = "--error-topic", description = "Error topic")
     protected String errorTopic;
     @CommandLine.Option(names = "--extra-input-topics", split = ",", description = "Additional named input topics",
-            converter = {UseDefaultConverter.class, StringListConverter.class})
+        converter = {UseDefaultConverter.class, StringListConverter.class})
```

Comment:
I just used downloaded our code style and the indent does not change so I think you misconfigured sth